my-project/templates/product_category/_edit_form.html.twig

```diff
@@ -19,19 +19,16 @@
 
 {{ include('product_category/_delete_form.html.twig') }}
 
-{{ form_start(imageForm, { 'attr' : { 'id' : 'AddImageForm' }}) }}
+{%  set route =  path('UploadAndAddImageToCategory', {'id': product_category.id}) %}
+{% set quote = "\"" %}
+{{ form_start(imageForm, { 'attr' : { 'onsubmit' : 'return onSubmitEditForm('~quote~route~quote~')' }}) }}
```

Comment:
@mSobonVirtua this should be passed as hidden input value or form attribute like data-someurl, not as method paramether